mam/api/BUILD

```diff
@@ -16,6 +17,15 @@ cc_library(
     ],
 )
 
+# Build command:
+# bazel build --spawn_strategy=local --define=workspace=$(bazel info workspace)  --define=output_base=$(bazel info output_base) --crosstool_top='@iota_toolchains//tools/emscripten:toolchain' --cpu=emscripten --host_crosstool_top='@bazel_tools//tools/cpp:toolchain' //mam/api:api.js
+
+emcc_binary(
```

Comment:
I would see this better in `mam/BUILD` and called `mam.js`, what do you think ?

Reply:
maybe.. anyways, done